### DISCUSSION TITLE
Will Java client be available anytime soon?

### DISCUSSION THREAD

**Discussion by cjacky475** [supabase/supabase#21185] [Questions]
Hello, I would like to use realtime futures from my spring boot app. I was not able to find any info about that. Are there any libraries or a roadmap for java? Cheers.

**Comment by GaryAustin1** [comment]
Most clients are community projects.  https://github.com/supabase-community
Java does not look like it is doing much.
Supabase does not usually comment on future roadmaps and clients are usually in that repository.
https://github.com/orgs/supabase/discussions/5#discussioncomment-80200  Is the main discussion for future clients.

I would plan on using the REST API's directly anytime in the near future.  IMO.
